auacm/angular/src/app/models/profile-picture.ts

```diff
@@ -0,0 +1,421 @@
+import { ElementRef } from '@angular/core';
+
+const SELECTION_RADIUS_MIN = 64;
+const PROFILE_PIC_DIMENSION = 256;
+
+const MAX_SELECTED_IMG_WIDTH: number = document.documentElement.clientWidth * 0.65;
```

Comment:
These need to be recalculated on resize if the user changes their client after the page loads. Better yet, you can make the width and height observables and subscribe to them.